symengine/visitor.h

```diff
@@ -209,8 +209,8 @@ class CoeffVisitor : public StopVisitor<CoeffVisitor> {
     }
 };
 
-RCP<const Basic> coeff(const Basic &b, const RCP<const Symbol> &x,
-        const RCP<const Integer> &n);
+RCP<const Basic> coef(const Basic &b, const RCP<const Symbol> &x,
+                      const RCP<const Integer> &n);
```

Comment:
What is the reason to just call it `coef` instead of `coeff`? Maple calls it coeff (http://www.maplesoft.com/support/help/maple/view.aspx?path=coeff).